source/robot_model/include/robot_model/Model.hpp

```diff
@@ -64,21 +49,16 @@ struct QPInverseVelocityParameters {
 class Model {
 private:
   // @format:off
-  std::shared_ptr<state_representation::Parameter<std::string>> robot_name_;///< name of the robot
-  std::shared_ptr<state_representation::Parameter<std::string>> urdf_path_; ///< path to the urdf file
+  std::string robot_name_;///< name of the robot
```

Comment:
nitpick: but if we have the format off, then we should at least align the comments imo

Reply:
or just turn it on ?